mw-webapp/src/service/ReportTest.ts

```diff
@@ -0,0 +1,88 @@
+// import {Report} from "src/model/report/Report";
+import {ReportDTO} from "src/model/report/ReportDTO";
+// import {WorkDone} from "src/model/report/workDone/WorkDone";
+// import {PlanForTomorrow} from "src/model/report/planForTomorrow/PLanForTomorrow";
+// import {Time} from "src/model/report/time/Time";
+// import {Unit} from "src/model/report/time/unit/Unit";
+import {ref, onValue, update, push, child} from "firebase/database";
+import {db} from "src/firebase";
+import {currentDate} from "src/utils/getDate";
+import {DayReport as DayReportDTO} from "src/model/firebaseCollection/DayReport";
+import {DayReport} from "src/model/businessModel/DayReport";
+// import {CurrentProblem} from "src/model/firebaseCollection/CurrentProblem";
+// import {JobDone} from "src/model/businessModel/JobDone";
+// import {PlanForNextPeriod} from "src/model/businessModel/PlanForNextPeriod";
+// import {CurrentProblem} from "src/model/businessModel/CurrentProblem";
+// import {Time} from "src/model/businessModel/time/Time";
+// import {Unit} from "src/model/report/time/unit/Unit";
+
+const reportDTOToBusinessConverter = (reportRaw: DayReportDTO) => new DayReport({
```

Comment:
Strange naming u don't have the model Business,so its unclear what is your target model

Reply:
This code and file will be removed after I fix render cells in table.